src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/Crucible.java

```diff
@@ -188,7 +188,7 @@ private void addLiquidLevel(@Nonnull Block block, boolean water) {
 
     private void placeLiquid(@Nonnull Block block, boolean water) {
         if (block.getType() == Material.AIR || block.getType() == Material.CAVE_AIR || block.getType() == Material.VOID_AIR) {
-            block.setType(water ? Material.WATER : Material.LAVA);
+            block.setType(water ? Material.WATER : Material.LAVA, false);
```

Comment:
Changes like these should usually be preceded by a comment explaining them with a reference to the issue.
Example:

```suggestion
            // Fixes #2903 - Cancel physics update to resolve weird overlapping
            block.setType(water ? Material.WATER : Material.LAVA, false);
```